src/makeRobotAccountant.js

```diff
@@ -1,12 +1,25 @@
-'use strict';
+"use strict";
 
 /**
- *
  * @return {function}
  */
-
 function makeRobotAccountant() {
-  // write code here
+  // Licznik wywołań, początkowo 0
+  let callCount = 0;
+
+  // Funkcja zwracana przez makeRobotAccountant
+  return function (a) {
+    return function (b) {
+      callCount++;
+
+      // Sprawdzamy, czy to czwarte lub kolejne wywołanie
+      if (callCount > 3 && callCount % 2 === 0) {
+        return "Bzzz... Error!";
+      } else {
+        return a + b;
+      }
+    };
+  };
```

Comment:
[CODE STYLE] - You don't need to name functions if you return them right away. Consider using anonymous functions instead to make the code cleaner. For example, `return function (b) { ... };`